web/components/Nav/Nav.tsx

```diff
@@ -21,6 +22,28 @@ export const Nav = ({ onFrontPage }: NavProps) => {
   const toggleMenu = () => setMenuOpened(!menuOpened);
   const closeMenu = () => setMenuOpened(false);
 
+  const BasicMenuItem = ({
+    urlPath,
+    label,
+  }: {
+    urlPath: string;
+    label: string;
+  }) => {
+    const isCurrentPage = urlPath === currentPath;
```

Comment:
Nit(personal preference): this constant might be inlined, which allows dropping the surrounding curlies and the explicit `return`.

Reply:
Yep, I actually had it like that initially. I have a slight preference to factoring out const/variable declarations to give them useful names, but it's not much of a difference here so I don't mind inlining it. (Number of lines is the same, there's more wrapping with the inlined constant.)

Reply:
1f1d8d248bb122d2a76e1daa783be7fe83721c31